### PR TITLE
Remove support for k3s and rke2 add-ons

### PR DESCRIPTION
**Pull request by emosbaugh**
#### What this PR does / why we need it:

Remove support for k3s and rke2 add-ons

#### Which issue(s) this PR fixes:

Fixes NONE

#### Does this PR introduce a user-facing change?

```release-note
* Removes support for the BETA K3s add-on and the BETA RKE2 add-on. It is recommended to use the [OpenEBS add-on](https://kurl.sh/docs/add-ons/openEBS#localpv) for the single-node LocalPV use case with kURL. For more information about this deprecation decision, see the [ADR](https://github.com/replicatedhq/kURL/blob/main/docs/arch/adr-007-deprecate-k3s-and-rke2.md).
```

#### Does this PR require documentation?

NONE

### CODE REVIEW

File: packages/kubernetes/1.18.13/conformance/Manifest [hunk 1]
```diff
@@ -1,2 +0,0 @@
-image conformance k8s.gcr.io/conformance:v1.18.13
```
Comment (rrpolanco) [inline packages/kubernetes/1.18.13/conformance/Manifest:1]: was this file intended to be removed?

Comment (emosbaugh) [reply packages/kubernetes/1.18.13/conformance/Manifest:1]: yes we dont have that version for kubeadm

**Review by rrpolanco — APPROVED** [review]
